certbot/storage.py

```diff
@@ -256,8 +259,26 @@ def __init__(self, config_filename, cli_config):
         self.live_dir = os.path.dirname(self.cert)
 
         self._fix_symlinks()
+        if update_symlinks:
+            self._update_symlinks()
         self._check_symlinks()
 
+    def __str__(self):
+        """Returns a string representation of the object."""
+        # import ipdb; ipdb.set_trace()
+        outdict = {"Lineage": self.lineagename,
+                   "Domains": " ".join(self.names())}
+        return "RenewableCert(%s)" % str(outdict)
```

Comment:
Plugins that we did not write could be using it.

I'd also prefer to hold off on the machine-readable part until we get outside requests/ use cases for it, unless you could point to any we've seen go by. The idea for this is to give users control over the interactions of their certs and their domains, both for selecting lineages to use and controlling which domains are in which cert. Which a human-readable list-certs will address, but a machine-readable won't.